modules/ding_wayf_dk/modules/ding_wayf_dk_create_user/templates/ding-wayf-dk-info-page.tpl.php

```diff
@@ -0,0 +1,13 @@
+<div class="default-layout">
+  <div class="layout-wrapper">
+    <h2 class="pane-title"><?php print render($title); ?></h2>
+    <div class="pane-content">
```

Comment:
Perhaps a comment explaining the relationship with panels would be an idea here.